File: Day 73/1491. Average Salary Excluding the Minimum and Maximum Salary.py
class Solution:
    def average(self, salary: List[int]) -> float:

        minIndex = maxIndex  = 0
        count = salary[0]

        for i in range(1, len(salary)):
            if(salary[i] > salary[maxIndex]):
                maxIndex = i
            if(salary[i] < salary[minIndex]):
                minIndex = i
            count += salary[i]


        return (count - salary[maxIndex] - salary[minIndex])/(len(salary) - 2)

File: Day 73/1491. Average Salary Excluding the Minimum and Maximum Salary.cpp
#include <iostream>
#include <vector>
using namespace std;

class Solution {
public:
    double average(vector<int>& salary) {
        
        int minIndex = 0, maxIndex = 0;
        double count = salary[0];

        for (int i = 1; i < salary.size() ; i++){
            if(salary[i] > salary[maxIndex])
                maxIndex = i;
            if(salary[i] < salary[minIndex])
                minIndex = i;
            count += salary[i];
        }

        return (count - salary[maxIndex] - salary[minIndex])/(salary.size() - 2);

    }
};

int main(){
  
  return 0;
}